src/test/java/com/bunq/sdk/model/generated/object/NotificationUrlTest.java

```diff
@@ -0,0 +1,238 @@
+            "/ChatMessageAnnouncement.json";
+    private static final String JSON_PATH_DRAFT_PAYMENT_MODEL = BASE_PATH_JSON_MODEL + "/DraftPayment.json";
+    private static final String JSON_PATH_MASTER_CARD_ACTION_MODEL = BASE_PATH_JSON_MODEL + "/MasterCardAction.json";
+    private static final String JSON_PATH_MONETARY_ACCOUNT_BANK_MODEL = BASE_PATH_JSON_MODEL +
+            "/MonetaryAccountBank.json";
+    private static final String JSON_PATH_PAYMENT_BATCH_MODEL = BASE_PATH_JSON_MODEL + "/PaymentBatch.json";
+    private static final String JSON_PATH_REQUEST_INQUIRY_MODEL = BASE_PATH_JSON_MODEL + "/RequestInquiry.json";
+    private static final String JSON_PATH_REQUEST_RESPONSE_MODEL = BASE_PATH_JSON_MODEL + "/RequestResponse.json";
+    private static final String JSON_PATH_SCHEDULE_PAYMENT_MODEL = BASE_PATH_JSON_MODEL + "/ScheduledPayment.json";
+    private static final String JSON_PATH_SCHEDULE_INSTANCE_MODEL = BASE_PATH_JSON_MODEL + "/ScheduledInstance.json";
+    private static final String JSON_PATH_SHARE_INVITE_BANK_INQUIRY_MODEL = BASE_PATH_JSON_MODEL +
+            "/ShareInviteBankInquiry.json";
+    private static final String JSON_PATH_SHARE_INVITE_BANK_RESPONSE_MODEL = BASE_PATH_JSON_MODEL +
+            "/ShareInviteBankResponse.json";
+
+    private static final Type NOTIFICATION_URL_TYPE = new TypeToken<NotificationUrl>(){}.getType();
+
+    private void executeNotificationUrlTest(
+            String expectedJsonFileName,
+            String classNameExpected,
+            String referencedObjectGetterName) throws FileNotFoundException, NoSuchMethodException,
+            InvocationTargetException, IllegalAccessException, ClassNotFoundException {
+        JsonReader jsonReader = new JsonReader(new FileReader(expectedJsonFileName));
+        NotificationUrl notificationUrl = new Gson().fromJson(jsonReader, NOTIFICATION_URL_TYPE);
+
+        assertNotNull(notificationUrl);
+        assertNotNull(notificationUrl.getObject());
+
+        Object model = notificationUrl.getObject().getClass().getDeclaredMethod(referencedObjectGetterName).invoke(
+                notificationUrl.getObject()
+        );
+        BunqModel referencedModel = notificationUrl.getObject().getReferencedObject();
+
+        assertNotNull(model);
+        assertNotNull(referencedModel);
+        assertTrue(Class.forName(classNameExpected).isInstance(referencedModel));
+    }
+
+    @Test
+    public void mutationModelTest() throws InvocationTargetException, FileNotFoundException,
+            ClassNotFoundException,
+            IllegalAccessException,
+            NoSuchMethodException {
+        executeNotificationUrlTest(
+                JSON_PATH_MUTATION_MODEL,
+                Payment.class.getName(),
+                GETTER_PAYMENT
+        );
+    }
+
+    @Test
+    public void bunqMeTabModelTest() throws InvocationTargetException, FileNotFoundException,
+            ClassNotFoundException,
```

Comment:
Our unit of indentation in Java is _two spaces_, and two more for hanging code. It seems the indents are wider here %)

Please take a look at the rest of the SDK code and format consistently with it :)

Reply:
My indention settings was on 4 😊 Changed to 2 and reformatted code.